cmd/agent/app/reporter/grpc/flags.go

```diff
@@ -39,6 +41,8 @@ func AddFlags(flags *flag.FlagSet) {
 	flags.Bool(collectorTLS, false, "Enable TLS.")
 	flags.String(collectorTLSCA, "", "Path to a TLS CA file. (default use the systems truststore)")
 	flags.String(collectorTLSServerName, "", "Override the TLS server name.")
+	flags.String(agentCert, "", "Path to a TLS client certificate file.")
+	flags.String(agentKey, "", "Path to a TLS client key file.")
```

Comment:
Could you please expand the flag descriptions (starting from `--tls`) to explain in more details what they enable? For example, what `collectorTLS` means after this PR, etc. Pretend someone is reading https://www.jaegertracing.io/docs/1.13/cli/#jaeger-collector-cassandra, is there enough information to understand what's going to happen with various combinations of the flags?